web/tests/pill_typeahead.test.js

```diff
@@ -140,7 +134,7 @@ for (const sub of subs) {
 }
 
 run_test("set_up", ({mock_template, override, override_rewire}) => {
-    override_typeahead_helper(override_rewire)
+    override_typeahead_helper(override_rewire);
```

Comment:
Some of these changes look like prettier fixups for the previous commit,

Reply:
Oops, I thought I'd caught all those. I'll do another linter pass on each commit once you've finished your review and I've addressed those comments.